ExampleApp/equityTicketConfig.json

```diff
@@ -0,0 +1,195 @@
+{
+	"fields": [
+		{
+			"apiParameter": "token",
+			"inputType": "hidden"
+		},
+		{
+			"apiParameter": "accountNumber",
+			"inputType": "accountSelection"
+		},
+		{
+			"apiParameter": "orderAction",
+			"label": "Action",
+			"inputType": "options",
+			"options": [
+				{
+					"value": "buy",
+					"label": "Buy"
+				},
+				{
+					"value": "sell",
+					"label": "Sell"
+				}
+			]
+		},
+		{
+			"apiParameter": "orderPrice",
+			"label": "Price",
+			"inputType": "options",
+			"options": [
+				{
+					"value": "market",
+					"label": "Market"
+				},
+				{
+					"value": "limit",
+					"label": "Limit"
+				},
+				{
+					"value": "stopLimit",
+					"label": "Stop Limit"
+				},
+				{
+					"value": "stop",
+					"label": "Stop"
+				}
+			]
+		},
+		{
+			"apiParameter": "orderQuantityType",
+			"label": "Quantity Type",
+			"inputType": "options",
+			"options": [
+				{
+					"value": "SHARES",
+					"label": "Shares"
+				},
+				{
+					"value": "TOTAL_VALUE",
+					"label": "USD",
+					"dependencies": [
+						{
+							"condition": "==",
```

Comment:
I'm not hot on `condition`. From a boolean logic perspective, this entire object is a condition, and this is the `operator`.

Reply:
Yah agreed, this was taken from something thrown together by Serge - try not to focus too much on the naming for now and try to evaluate on:
1. How easy it would be to create new clients (for both us on different platforms and third parties creating custom tickets)
1. Does it cover all of the cases we currently have (reuse same VC for equity/crypto/etc)
1. Do we prefer this style of dependencies/prerequisites lists vs a tree structure